packages/dapp-kit-react-privy/src/DAppKitPrivyProvider/DAppKitPrivyProvider.tsx

```diff
@@ -0,0 +1,88 @@
+import { ReactNode } from "react";
+import { PrivyProvider as BasePrivyProvider } from "@privy-io/react-auth";
+import { DAppKitProvider } from "@vechain/dapp-kit-react";
+
+type Props = {
+  children: ReactNode;
+  appId: string;
+  clientId: string;
+  appearance: {
+    theme: "dark" | "light";
+    accentColor: `#${string}`;
+    loginMessage: string;
+    logo: string;
+  };
+  embeddedWallets: {
+    createOnLogin: "all-users";
+  };
+  loginMethods: (
+    | "wallet"
+    | "email"
+    | "sms"
+    | "google"
+    | "twitter"
+    | "discord"
+    | "github"
+    | "linkedin"
+    | "spotify"
+    | "instagram"
+    | "tiktok"
+    | "apple"
+    | "farcaster"
+    | "telegram"
+  )[];
+  smartAccountConfig: {
+    nodeUrl: string;
+    delegatorUrl: string;
+    accountFactoryAddress: string;
+  };
+  dappKitConfig: {
```

Comment:
Can we import the dapp kit config directly, instead of creating a new version of it?

The `genesis` is a different type for example